### PR TITLE
refactor(Rest): `xID` to `xId`

### PR DESCRIPTION
**Pull request by ckohen**
**Please describe the changes this PR makes and why it should be merged:**

**Status and versioning classification:**

- I know how to update typings and have done so, or typings don't need updating
- This PR includes breaking changes (methods removed or renamed, parameters moved or removed)

### CODE REVIEW

**Review by kyranet — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

File: packages/rest/src/lib/CDN.ts [hunk 1]
```diff
@@ -14,22 +14,22 @@ export class CDN {
 
 	/**
 	 * Generates an app asset URL for a client's asset.
-	 * @param clientID The client ID that has the asset
+	 * @param clientId The client id that has the asset
```
Comment (jonahsnider) [inline packages/rest/src/lib/CDN.ts:17]: The comment here shouldn't be updated, just the parameter name should be changed.

Comment (monbrey) [reply packages/rest/src/lib/CDN.ts:17]: `id` is correct, not ID. It's short for identifier, lowercase, not an acronym of Identity Document.

**Comment by iCrawl** [comment]
I'm just not gonna ask how we lost 0.2% of coverage on that.

File: packages/rest/src/lib/CDN.ts (hunk 1 above)
Comment (jonahsnider) [reply packages/rest/src/lib/CDN.ts:17]: Seems like it's still supposed to be uppercased:
dictionary definition https://www.merriam-webster.com/dictionary/id

**Review by iCrawl — APPROVED** [review]

**Pull request merged**